fdbclient/Schemas.cpp

```diff
@@ -981,6 +981,10 @@ const KeyRef JSONSchemas::statusSchema = R"statusSchema(
          "expired_age": 0,
          "oldest_id_version": 0,
          "oldest_id_age": 0
+      },
+	  "version_epoch":{
```

Comment:
BTW, this new field should also be added to `mr-status-json-schemas.rst.inc` file. It looks like some new fields, e.g., `idempotency_ids`, `metacluster` were missing as well, not sure if that's intentional or not.

Reply:
Never mind. I was looking at `release-7.2` branch.